tools/service.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This script can be used by cron to auto restart Datalad-Registry service upon
+# the host's reboot.
+
+set -eu
+umask 077
+
+# Check if the path to the Datalad-Registry repo is provided
+if [ -z "$1" ]; then
```

Comment:
due to `set -eu` -- this would fail AFAIK... if it does, remove `set -eu` above or
```suggestion
if [ -z "${1:-}" ]; then
```